services/mailservice/mail.go

```diff
@@ -140,15 +157,15 @@ func ConnectToSMTPServerAdvanced(connectionInfo *SmtpConnectionInfo) (net.Conn,
 	return conn, nil
 }
 
-func ConnectToSMTPServer(config *model.Config) (net.Conn, error) {
+func ConnectToSMTPServer(config *SMTPConfig) (net.Conn, error) {
 	return ConnectToSMTPServerAdvanced(
 		&SmtpConnectionInfo{
```

Comment:
I am wondering if there is a need to duplicate the config in two structs? It seems like `SmtpConnectionInfo` already contains what we need. If we are exposing `SMTPConfig` to the outside world, can we just clean up `SmtpConnectionInfo` and use only one config struct throughout?

Reply:
Yes, that makes a lot of sense, for sure we can make it simpler.

Reply:
done